collect_app/src/androidTest/java/org/odk/collect/android/regression/FillBlankInvalidFormTest.java

```diff
@@ -0,0 +1,45 @@
+package org.odk.collect.android.regression;
+
+import android.Manifest;
+
+import androidx.test.rule.GrantPermissionRule;
+import androidx.test.runner.AndroidJUnit4;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.junit.runner.RunWith;
+import org.odk.collect.android.support.CopyFormRule;
+import org.odk.collect.android.support.ResetStateRule;
+import org.odk.collect.android.support.pages.MainMenuPage;
+
+//Issue NODK-244
+@RunWith(AndroidJUnit4.class)
+public class FillBlankInvalidFormTest extends BaseRegressionTest {
+    @Rule
+    public RuleChain copyFormChain = RuleChain
+            .outerRule(GrantPermissionRule.grant(
+                    Manifest.permission.READ_EXTERNAL_STORAGE,
+                    Manifest.permission.WRITE_EXTERNAL_STORAGE,
+                    Manifest.permission.READ_PHONE_STATE)
+            )
+            .around(new ResetStateRule())
+            .around(new CopyFormRule("invalid-events.xml"))
+            .around(new CopyFormRule("asasas.xml"))
+            .around(new CopyFormRule("setlocation-and-audit-location.xml"))
+            .around(new CopyFormRule("setlocation-action-instance-load.xml"));
+
+
+    @Test
+    public void forms_shouldNotBeVisibleOnFormList() {
```

Comment:
This name could be improved. I guess the shouldn't be visible because they are broken? If so something like:
`brokenForms_shouldNotBeVisibleOnFOrmList` would be much clearer.